src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNT = Colors.values().length;
+    private final Random random = new Random();
```

Comment:
Why final? In this case, nothing critical, but in the future it is better not to do this